src/java.desktop/windows/classes/sun/awt/windows/TranslucentWindowPainter.java

```diff
@@ -66,14 +65,10 @@ abstract class TranslucentWindowPainter {
     protected WWindowPeer peer;
 
     // REMIND: we probably would want to remove this later
-    @SuppressWarnings("removal")
     private static final boolean forceOpt  =
-        Boolean.parseBoolean(AccessController.doPrivileged(
-            new GetPropertyAction("sun.java2d.twp.forceopt", "false")));
-    @SuppressWarnings("removal")
+        Boolean.getBoolean(System.getProperty("sun.java2d.twp.forceopt", "false"));
     private static final boolean forceSW  =
-        Boolean.parseBoolean(AccessController.doPrivileged(
-            new GetPropertyAction("sun.java2d.twp.forcesw", "false")));
+        Boolean.getBoolean(System.getProperty("sun.java2d.twp.forcesw", "false"));
```

Comment:
This seems to mistakenly combine `Boolean.getBoolean()` and `System.getProperty()`, instead of only using the former or using `Boolean.parseBoolean()` with the latter. Same issue above for `forceOpt`.

Reply:
Skara hasn't removed the above "Terms of User" banner, but I'm pretty sure that @msternermxy is right, and that this is not equivalent to the original.

It should either be `Boolean.parseBoolean(System.getProperty("sun.java2d.twp.forcesw", "false"));` or else  `Boolean.parseBoolean("sun.java2d.twp.forcesw");` -- the latter being OK because false is the default for a property that is not set. Same for `forceOpt`.

Reply:
oops. he is right, Kevin you have a mistake

"or else Boolean.parseBoolean("sun.java2d.twp.forcesw");" you mean Boolean.getBoolean(..) !

I think the first change made was to
Boolean.parseBoolean(System.getProperty("sun.java2d.twp.forceopt", "false"));
and I suggested changing to the simpler Boolean.getBoolean() but I meant as a replacement for BOTH calls.
ie to
Boolean.getBoolean("sun.java2d.twp.forcesw")
I didn't notice the mistake on re-review. I guess I too quickly assumed updates were likely to be corrections.

Reply:
OK done...